modules/swagger-codegen/src/main/resources/php/Configuration.mustache

```diff
@@ -123,51 +109,6 @@ class Configuration
     protected $tempFolderPath;
 
     /**
-     * Indicates if SSL verification should be enabled or disabled.
```

Comment:
Generated Api classes are now only creating psr compatible requests and handle responses. How this stuff is transferred is problem of the client we are trying to separate from. So ssl or proxy has to be supported and configured in provided client. For example:

guzzle client:
```
$client = new \Http\Adapter\Guzzle6\Client(['verify_ssl' => false]); //in guzzle its on by default
$apiInstance = new Swagger\Client\Api\PetApi($client)); //its enabled by default in guzzle afaik
```
curl client:
```
$client = new Http\Client\Curl\Client(null, null, ['CURLOPT_SSL_VERIFYHOST' => true]);
$apiInstance = new Swagger\Client\Api\PetApi($client)); //its enabled by default in guzzle afaik
```